SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditor.h

```diff
@@ -17,7 +17,14 @@ class SPATIALGDKEDITOR_API FSpatialGDKEditor
 	{
 	}
 
-	bool GenerateSchema(bool bFullScan);
+	enum SchemaGenerationMethod
```

Comment:
Prefix this with E please!
https://docs.unrealengine.com/en-US/Programming/Development/CodingStandard/index.html#namingconventions